Exercise-5/VideoPlayerDemo.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Bridge Pattern
+ */
+interface VideoPlatform{
+	public String playOn();
+}
+class Netflix implements VideoPlatform{
+	@Override
+	public String playOn() {
+		return "Netflix";
+	}
+}
+class HotStar implements VideoPlatform{
+	@Override
+	public String playOn() {
+		return "Hotstar";
+	}	
+}
+
+
+abstract class Video{
+	VideoPlatform platform;
+	public Video(VideoPlatform plat) {
+		platform=plat;
+	}
+	abstract public void play(); 
+}
+
+
+class Movie extends Video{
```

Comment:
You can have a small space here between Video and { - just to make the code look god